aws/src/integration/java/org/apache/iceberg/aws/s3/S3FileIOTest.java

```diff
@@ -202,8 +201,7 @@ public void testACL() throws Exception {
 
   @Test
   public void testClientFactorySerialization() throws Exception {
-    S3FileIO fileIO = new S3FileIO();
-    fileIO.initialize(Maps.newHashMap());
+    S3FileIO fileIO = new S3FileIO(clientFactory::s3);
```

Comment:
Does this change need to be in this PR?

Reply:
Otherwise it will throw exception because glue lock is not found when initializing `AwsProperties`. Changing the constructor resolves the issue.